frontend/components/Domain/Recipe/RecipePrintView.vue

```diff
@@ -83,19 +83,43 @@
         </div>
       </section>
     </div>
+
+    <!-- Nutrition -->
+    <div v-if="preferences.showNutrition">
+      <v-card-title class="headline pl-0"> {{ $t("recipe.nutrition") }} </v-card-title>
+
+      <v-divider v-if="hasNutrition" class="grey my-4"></v-divider>
```

Comment:
this will never get shown because `hasNutrition` will always be negative as props.recipe.nutrition.length is undefined. 

I also think the divider is not necessairy 
```suggestion
```